### ISSUE TITLE
Remove plugin gulp-spawn-mocha from the blacklist

### ISSUE THREAD

**Issue by KenPowers** [gulpjs/plugins#42]
I do not believe it is fair to consider [`gulp-spawn-mocha`](https://github.com/KenPowers/gulp-spawn-mocha) a duplicate of [`gulp-mocha`](https://github.com/sindresorhus/gulp-mocha). I am very clear in my documentation (updated for version 0.1.2) about when it is appropriate to use either plugin. I created `gulp-spawn-mocha` because I was running into a very specific issue -- the fix for said issue does not fit in to the existing `gulp-mocha` plugin and actually has some very nice side-effects which can help people in other edge cases.

By starting `mocha` in a separate process I am offering completely separate functionality from what the existing `gulp-mocha` provides. Tests run in their own clean environment every time. Everything your tests do is kept separate from the build system and the `gulp` process cannot crash because something in `mocha` crashed (this is different from catching exceptions, as highlighted in my documentation).

I also allow people to use their own versions of `mocha`. This means that if they're relying on a specific version of `mocha` for some reason (maybe an early beta, maybe an old version, maybe a [fork with bug fixes for certain platforms](https://github.com/airportyh/mocha/tree/windowsfix)) that they can still use their special `mocha` along with `gulp`. This isn't possible in the existing `gulp-mocha` plugin and although it's not extremely common, it is common enough that I do see a place for my plugin to fill the need of certain users.

I am not asking that people use my plugin instead of `gulp-mocha`, that's just silly and I state in my documentation that `gulp-mocha` is *probably* what you're looking for. My plugin simply does some things (e.g., clearing globals and defined properties) which are simply not possible in the existing `gulp-mocha`. Even if you don't write code which uses globals or defined properties that doesn't mean you aren't relying on code that does use these features. This is particularly problematic when watching for changes and rerunning tests.

In conclusion I'd like to say that calling `gulp-spawn-mocha` a duplicate of `gulp-mocha` is like calling [`gulp-ruby-sass`](https://github.com/sindresorhus/gulp-ruby-sass) a duplicate of [`gulp-sass`](https://github.com/dlmanning/gulp-sass). Yes, there is a lot of overlapping functionality; however, the process in which everything is accomplished is different enough to have an impact on the end user.

**Comment by KenPowers** [comment]
Closing. See KenPowers/gulp-spawn-mocha#3.
